New package: DistributedWorkflows v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: DistributedWorkflows
- Repository: https://github.com/FiroozehDastur/DistributedWorkflows.jl
- Created by: @FiroozehDastur
- Version: v0.2.0
- Commit: d347ca09a23543c27af85acd13ef1b89b246be97
- Git reference: v0.2.0
- Description: A Julia interface to a distributed task-based workflow management system
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
A Julia interface to a distributed task-based workflow management system
`````
<!-- END RELEASE NOTES -->

goerz commented:
[noblock] Just to double-check: `Documenter` is probably not actually a dependency of your project, right? Usually, it is only required for building the documentation. It is important to keep the dependencies for the actual package in your `Project.toml` as minimal as possible. You do not want to include any packages that are used only for testing or for building the documentation. Those go in `test/Project.toml` (or the `[extras]` section of the main `Project.toml`) and in `docs/Project.toml`, respectively.

FiroozehDastur commented:
[no block] Hi... thanks for pointing that out. I was not sure after reading the documentation, if I should have test/docs dependencies in the main `Project.toml` or not. I will remove the unused/unnecessary dependencies. 

Can I push the changes to this PR? or should I just update the main branch of the repo? Thanks again for pointing out the redundancy of deps

GunnarFarneback commented:
If you update your main branch **and keep the version number**, a new registration will update this PR.

JuliaHubRegistrator committed:
New package: DistributedWorkflows v0.2.0

UUID: e6614011-adc0-4f34-baa6-5a168cd5013f
Repo: https://github.com/FiroozehDastur/DistributedWorkflows.jl.git
Tree: 98ce3a63276d2ba7967073d60dfa07f5e9a1027a

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
